src/module/server-instance/ServerInstanceCtrl.ts

```diff
@@ -0,0 +1,85 @@
+import {deepGet} from "../../common/utils/Utils";
+import {ServerService} from "../../services/server/ServerService";
+import {ServerAddress} from "../../services/server/ServerAddress";
+import {IServerAddress} from "../../services/server/IServerAddress"
+import {IServer} from "../../services/server/IServer"
+import {IStateParamsService} from "angular-ui-router";
+import {IIntervalService} from "angular";
+import {IServerGroup} from "../../services/server-group/IServerGroup";
+import {InstanceMemoryChart} from "./MemoryChart";
+
+export class ServerInstanceCtrl {
+    static $inject: string[] = ["$interval", "$stateParams", "serverService", "coord", "serverGroup", "serverInstance"];
+
+    threadCount: number;
+    threadPeakCount: number;
+    threadDaemonCount: number;
+
+    directBufferPoolCount: number;
+    directBufferPoolMemoryUsed: number;
+
+    mappedBufferPoolCount: number;
+    mappedBufferPoolMemoryUsed: number;
+
+    nodeStats: any;
+    chart:InstanceMemoryChart;
+
+    constructor(private $interval: IIntervalService, private $stateParams: IStateParamsService,
```

Comment:
There's no need to pass $stateParams into the controller as you are only using it to get information that has already been passed to the controller.